lib/controller.js

```diff
@@ -506,6 +506,10 @@ controller.update = function(opts) {
               // Once we have the current SHA, provide the version
               .then(function(currentSHA) {
                 return resolve(updates.findBuild(builds, 'sha', currentSHA).version);
+              })
+              .catch(function () {
+                console.log('Could not fetch current tessel version. Perhaps use --force?');
```

Comment:
Since we already have a `tessel` open at this point, it needs to be completely closed. This needs to be something like: 

```js
logs.warn('Could not fetch current tessel version. Perhaps use --force?');

return controller.closeTesselConnections([tessel]).then(resolve);
```